src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] data = readFileText(fromFileName);
+        String report = calculateData(data);
+        writeToFile(toFileName, report);
+    }
+
+    private String[] readFileText(String fromFileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            StringBuilder stringBuilder = new StringBuilder();
```

Comment:
Please don't name 'container' data types like this (List list,Map map etc), name them after what they store :) it's an opportunity to make code more readable. stringBuilder doesn't tell us anything new

Reply:
not fixed
use `fileContent `name for example